lacework/resource_lacework_data_export_rule.go

```diff
@@ -0,0 +1,201 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkDataExportRule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkDataExportRuleCreate,
+		Read:   resourceLaceworkDataExportRuleRead,
+		Update: resourceLaceworkDataExportRuleUpdate,
+		Delete: resourceLaceworkDataExportRuleDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkDataExportRule,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "The name of the data export rule",
+				Required:    true,
+			},
+			"profile_versions": {
+				Type:        schema.TypeList,
+				Required:    true,
+				Description: "List of profile versions",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					StateFunc: func(val interface{}) string {
+						return strings.TrimSpace(val.(string))
+					},
+				},
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The type of the data export rule",
+				Default:     "Dataexport",
+			},
+			"integration_ids": {
+				Type:        schema.TypeList,
+				Required:    true,
+				Description: "List of integration ids",
```

Comment:
This description is so vague, our API schema says this:
```
      "description": "The alert channels for the rule to use.",
```
Can we do a better description and documentation. Make it so that customers can use it without having to read more documentation or APIs.